docs/scripts_md/Database.md

```diff
@@ -0,0 +1,141 @@
+# NAME
+
+NeuroDB::Database -- Provides a set of methods to run SQL statements on a database
+
+# SYNOPSIS
+
+    use NeuroDB::Database;
+    use TryCatch;
+
+    my $db = NeuroDB::Database->new(
+        userName     => 'user',
+        databaseName => 'my_db',
+        hostName     => 'my_hostname',
+        password     => 'pwd'
+    );
+
+    try {
+        $db->connect();
+    } catch(NeuroDB::DatabaseException $e) {
+        die sprintf(
+            "User %s failed to connect to %s on %s: %s (error code %d)\n",
+            'user',
+            'my_db',
+            'my_hostname',
+            $e->errorMessage,
+            $e->errorCode
+        );
+    }
+
+    .
+    .
+    .
+
+    try {
+        $db->pselect(
+            'SELECT * FROM candidate WHERE pscid = ?', 'MTL135'
+        );
+    } catch(NeuroDB::DatabaseException $e) {
+        die sprintf(
+            "SELECT on candidate table failed: %s (error code=%d)",
+            $e->errorMessage,
+            $e->errorCode
+        );
+    }
+
+# DESCRIPTION
+
+This class provides the basic SELECT, INSERT, UPDATE and DELETE methods
```

Comment:
Nitpicking, should `SELECT`, `INSERT`, `UPDATE` and `DELETE` be in code format?

Reply:
OK